hass_nabucasa/voice_data.py

```diff
@@ -0,0 +1,1188 @@
+"""
+Available voices for TTS.
+
+Automatically generated file, do not edit this file directly.
+Run python3 -m scripts/update_voice_data.py to update this file.
+"""
+
+TTS_VOICES: dict[str, dict[str, dict]] = {
+    "af-ZA": {
+        "AdriNeural": {"name": "Adri"},
+        "WillemNeural": {"name": "Willem"},
+    },
+    "am-ET": {
+        "AmehaNeural": {"name": "Ameha"},
+        "MekdesNeural": {"name": "Mekdes"},
+    },
+    "ar-AE": {
+        "FatimaNeural": {"name": "Fatima"},
+        "HamdanNeural": {"name": "Hamdan"},
+    },
+    "ar-BH": {
+        "AliNeural": {"name": "Ali"},
+        "LailaNeural": {"name": "Laila"},
+    },
+    "ar-DZ": {
+        "AminaNeural": {"name": "Amina"},
+        "IsmaelNeural": {"name": "Ismael"},
+    },
+    "ar-EG": {
+        "SalmaNeural": {"name": "Salma"},
+        "ShakirNeural": {"name": "Shakir"},
+    },
+    "ar-IQ": {
+        "BasselNeural": {"name": "Bassel"},
+        "RanaNeural": {"name": "Rana"},
+    },
+    "ar-JO": {
+        "SanaNeural": {"name": "Sana"},
+        "TaimNeural": {"name": "Taim"},
+    },
+    "ar-KW": {
+        "FahedNeural": {"name": "Fahed"},
+        "NouraNeural": {"name": "Noura"},
+    },
+    "ar-LB": {
+        "LaylaNeural": {"name": "Layla"},
+        "RamiNeural": {"name": "Rami"},
+    },
+    "ar-LY": {
+        "ImanNeural": {"name": "Iman"},
+        "OmarNeural": {"name": "Omar"},
+    },
+    "ar-MA": {
+        "JamalNeural": {"name": "Jamal"},
+        "MounaNeural": {"name": "Mouna"},
+    },
+    "ar-OM": {
+        "AbdullahNeural": {"name": "Abdullah"},
+        "AyshaNeural": {"name": "Aysha"},
+    },
+    "ar-QA": {
+        "AmalNeural": {"name": "Amal"},
+        "MoazNeural": {"name": "Moaz"},
+    },
+    "ar-SA": {
+        "HamedNeural": {"name": "Hamed"},
+        "ZariyahNeural": {"name": "Zariyah"},
+    },
+    "ar-SY": {
+        "AmanyNeural": {"name": "Amany"},
+        "LaithNeural": {"name": "Laith"},
+    },
+    "ar-TN": {
+        "HediNeural": {"name": "Hedi"},
+        "ReemNeural": {"name": "Reem"},
+    },
+    "ar-YE": {
+        "MaryamNeural": {"name": "Maryam"},
+        "SalehNeural": {"name": "Saleh"},
+    },
+    "as-IN": {
+        "PriyomNeural": {"name": "Priyom"},
+        "YashicaNeural": {"name": "Yashica"},
+    },
+    "az-AZ": {
+        "BabekNeural": {"name": "Babek"},
+        "BanuNeural": {"name": "Banu"},
+    },
+    "bg-BG": {
+        "BorislavNeural": {"name": "Borislav"},
+        "KalinaNeural": {"name": "Kalina"},
+    },
+    "bn-BD": {
+        "NabanitaNeural": {"name": "Nabanita"},
+        "PradeepNeural": {"name": "Pradeep"},
+    },
+    "bn-IN": {
+        "BashkarNeural": {"name": "Bashkar"},
+        "TanishaaNeural": {"name": "Tanishaa"},
+    },
+    "bs-BA": {
+        "GoranNeural": {"name": "Goran"},
+        "VesnaNeural": {"name": "Vesna"},
+    },
+    "ca-ES": {
+        "AlbaNeural": {"name": "Alba"},
```

Comment:
@ludeeus We could simplify this data format and impact on memory if we replace `{"name": "Alba"}` with `"Alba"` and check if it's a string when loading. WDYT? Most users are only going to look at a single language anyway.

Reply:
As most of them also only have a name, yes that makes sense to me 👍